wandb/sdk/wandb_settings.py

```diff
@@ -347,7 +347,7 @@ class Settings:
     _cli_only_mode: bool  # Avoid running any code specific for runs
     _config_dict: Config
     _cuda: str
-    _debug_log: str
+    _tracelog: str
```

Comment:
Please keep it in the alphabetic order :)

Reply:
oops... of course.. fixed.